app/browser-sync.ts

```diff
@@ -93,6 +95,16 @@ export class TargetLocatorSync {
     exec(this.targetLocator.window(nameOrHandle));
   }
 
+  frame(nameOrElement: ElementFinderSync | string): void {
```

Comment:
Yup, it's similar. I think either option is fine, but that we're in a slightly weird place right now:

1. Just accept an element finder. Nothing wrong with this; it's simpler for the caller to select the element on their own.
2. In any case where a method provides "built-in" selection, it seems like it shouldn't just provide a single way to do it (in this case, looking up by name). So, if we're going to provide "element lookup services" in a method, I think it should accept any form of looking up the element.

PS: If this is the same API used by protractor, feel free to ignore this, as keeping the API consistent is also a valid argument. Although selecting frames by name only is a remnant of the web before iframes existed... :0